packages/repo/src/sync.ts

```diff
@@ -1,33 +1,118 @@
-import * as auth from '@atproto/auth'
-import { CID } from 'multiformats/cid'
-import { RepoStorage } from './storage'
-import { DataDiff } from './mst'
+import { MemoryBlockstore, RepoStorage } from './storage'
+import DataDiff from './data-diff'
 import Repo from './repo'
 import * as verify from './verify'
+import { DidResolver } from '@atproto/did-resolver'
+import * as util from './util'
+import { CommitData, RecordWriteOp, RepoContents } from './types'
+import { CID } from 'multiformats/cid'
+import CidSet from './cid-set'
+import { MissingBlocksError } from './error'
```

Comment:
```suggestion
import { CID } from 'multiformats/cid'
import { DidResolver } from '@atproto/did-resolver'
import { MemoryBlockstore, RepoStorage } from './storage'
import DataDiff from './data-diff'
import Repo from './repo'
import * as verify from './verify'
import * as util from './util'
import { CommitData, RecordWriteOp, RepoContents } from './types'
import CidSet from './cid-set'
import { MissingBlocksError } from './error'
```

Reply:
A little hard to see in the suggestion, but I just pulled `multiformats/cid` and `@atproto/did-resolver` up to the top.